lib/generators/draper/templates/application_decorator.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationDecorator < Draper::Decorator
```

Comment:
I removed `delegate_all` from this template.  The belief is that certain issues and bad practices may stem from having this line in all decorators by default.  Implementers should be encouraged to whitelist their delegations to minimize this risk.